android/build.gradle

```
@@ -5,7 +5,7 @@ buildscript {
         buildToolsVersion = "29.0.2"
```

React Native upgrade helper ( https://react-native-community.github.io/upgrade-helper/?from=0.64.1&to=0.66.2 ) recommends 30.0.2 here.
```suggestion
        buildToolsVersion = "30.0.2"
```